src/util/QdmTestCaseHelper.ts

```diff
@@ -0,0 +1,23 @@
+import { TestCase } from "@madie/madie-models";
+import * as _ from "lodash";
+import { QDMPatient } from "cqm-models";
+import { ObjectID } from "bson";
+import { v4 as uuidv4 } from "uuid";
+
+export function cloneTestCase(testCase: TestCase) {
+  if (_.isNil(testCase)) {
+    return;
+  }
+  const clonedTestCase = _.cloneDeep(testCase);
+  if (!_.isEmpty(clonedTestCase.json)) {
+    const qdmPatient = new QDMPatient(JSON.parse(clonedTestCase?.json));
+    qdmPatient._id = new ObjectID().toString();
```

Comment:
we faced some issues with assigning value to `id/_id` in the past. assigning value to `_id` also updates `id`, right?  or maybe not because this is stored as json string. might have to update `id `as well

Reply:
when I log out the QDMPatient object, I don't see an "id" field, only "_id"

Reply:
I think that's right. thanks Nate!